### DISCUSSION TITLE
[Feature Request] Easily identify what dashboards and panels use a given data source

### DISCUSSION THREAD

**Discussion by CloudPlatformer** [grafana/grafana#42976] [Data Sources]
In addition to the default Settings tab of Data Sources currently (as of v8.3.1), could there be another tab to list what Dashboards (and which panels of those Dashboards) within the Grafana organisation are using the data source?

This would be very useful, especially when a change must be made to all panels that use a given data source.

An example of such a change is when panels that used Azure Monitor data sources for Azure Metrics Logs queries had to be edited and then saved following Azure Monitor data sources no longer supporting different credentials for Metrics and Logs:
'Panels will be migrated to use the new resource-centric backend when you first edit and save them.' on https://github.com/grafana/grafana/releases/tag/v8.1.0-beta1

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82968. Feel free to continue the discussion around this there. Thank you!
